redhat-developer/rhosak-ui
# feat: add check for terms and condition when creating an instance

**What this PR does / why we need it**:

Checks the terms and conditions API to see if the user has accepted the T&S or if there is a new one to accept.

**Verification steps** 

* Create a new user
* click on Create Kafka instance
* a dialog asking to accept the terms and condition should be displayed
